R/strategy_support.R

```diff
@@ -47,7 +47,7 @@ environment_type <- function(type) {
          stop("Unknown type ", type))
 }
 
-make_environment <- function(type, ...) {
+make_environment <- function(type = "FF16", ...) {
```

Comment:
I wonder if we should remove the default type, to force that type needs to be specified?